gateway-service-remoteconfig/pom.xml

```diff
@@ -38,6 +38,10 @@
             <groupId>org.apache.knox</groupId>
             <artifactId>gateway-spi</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Is this just for `ConfigurationException`?

Reply:
Yes. If there is an issue with the supplied JAAS config it is a `ConfigurationException`, right?
Moreover `ConfigurationException` is not a checked one so that I did not have to modify the API. I think it would be an exaggeration to create a new type of exception just for this purpose if we already have to proper one. Agreed?

Reply:
Yea just checking that we didn't pull in a new dependency for another reason.